src/postgres/src/test/regress/yb_pg_join_schedule

```diff
@@ -6,9 +6,8 @@
 # Dependencies for `yb_pg_join`
 test: yb_pg_text
 test: yb_pg_int4
-test: yb_pg_int2
 test: yb_pg_int8
-test: yb_pg_float8
+test: yb_pg_create_join
```

Comment:
`create_join` is not in `serial_schedule`.  Plus, it looks like some workaround.  Is this WIP and you need to now add back `yb_pg_int2` and `yb_pg_float8` since you added `DROP TABLE` back?

Reply:
To clarify, either change it to `yb_create_join` since it's not an original pg test OR (preferrably) remove it and bring back `yb_pg_int2` and `yb_pg_float8` if you can.